app-backend/api/src/main/scala/exports/Routes.scala

```diff
@@ -78,10 +80,15 @@ trait ExportRoutes extends Authentication
   def createExport: Route = authenticate { user =>
     entity(as[Export.Create]) { newExport =>
       authorize(user.isInRootOrSameOrganizationAs(newExport)) {
-        onSuccess(write[Export](Exports.insertExport(newExport, user))) { export =>
-          kickoffProjectExport(export.id)
-          complete(export)
-        }
+        (for {
+          ei <- newExport.exportOptions.as[ExportOptions].toOption
+        } yield {
+          onSuccess(write[Export](Exports.insertExport(newExport, user))) { export =>
+            kickoffProjectExport(export.id)
+            complete(export)
+          }
+        }).getOrElse(
+          complete((StatusCodes.BadRequest, "Parameter 'exportOptions' could not be decoded to type 'ExportOptions'")))
```

Comment:
```scala
scala> case class Foo(x: Int)
scala> case class Bar(f: Foo)
scala> val badString = """{"f":{"y":1}}"""
scala> decode[Bar](badString)
Either[io.circe.Error,Bar] = Left(DecodingFailure(Attempt to decode value on failed cursor, List(DownField(x), DownField(f))))
```

The `Left` there is the same error that the helpful json error parser parses to tell people "missing field: $field". It only tells one field at a time as it currently stands, but it would at least let people know a field at a time that's wrong instead of trying to find it on their own.